tools/bazel

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash -e
```

Comment:
Did we borrow this from somewhere? If so, make sure to include a link

Reply:
Sure, I will.